redis/Cargo.toml

```diff
@@ -54,10 +54,16 @@ r2d2 = { version = "0.8.8", optional = true }
 # Only needed for cluster
 crc16 = { version = "0.4", optional = true }
 rand = { version = "0.8", optional = true }
+derivative = { version = "2.2.0", optional = true }
+
 # Only needed for async_std support
 async-std = { version = "1.8.0", optional = true}
 async-trait = { version = "0.1.24", optional = true }
-derivative = { version = "2.2.0", optional = true }
+# To avoid conflicts, backoff-async.version != backoff-tokio.version so we could run tests with --all-features
```

Comment:
@nihohit I know you're going to hate it, but I couldn't find any other solution that works. When running with --all-features, if both backoff-async and backoff-tokio share the same version, it failes with:
```
the crate `redis v0.23.0 (/home/ubuntu/babushka/submodules/redis-rs/redis)` depends on crate `backoff v0.4.0` multiple times with different names

Reply:
https://amzn-aws.slack.com/archives/C017S7L2430/p1690976940063639